### PR TITLE
Scripts for downloading HAR dataset and organising it into train-val-test files

### PR DESCRIPTION
**Pull request by chogba**
Adding scripts for downloading UCI HAR dataset from the internet and organising it into train-val-test .npy splits. Every split has a separate set of users, and the same user data is not part of two splits. Running fetch_har.py downloads and extracts the dataset and generates train-val-test .npy files in the extracted directory.

### CODE REVIEW

**Comment by metastableB** [comment]
@chogba  Can you rename the PR and add more details as to what this PR has ?

File: tf/examples/EMI-RNN/helpermethods.py [hunk 1]
```diff
@@ -1,70 +1,73 @@
 # creates x_train, y_train, x_test, y_test, x_val, y_val npy files. val set is created after cutting out a user provided fraction from train set.
```
Comment (metastableB) [inline tf/examples/EMI-RNN/helpermethods.py]: Can you please remove this line, add a proper comment and our Privacy policy line here ?

Comment (chogba) [reply tf/examples/EMI-RNN/helpermethods.py]: @metastableB done

**Comment by chogba** [comment]
@metastableB please check

**Comment by chogba** [comment]
After extraction, ./HAR/UCI HAR Dataset/train/Inertial Signals contains 7352 raw sensor values of ax,ay,az,gx,gy,gz cofrresponding to one window (128 timesteps long) recorded from the training subjects. Similarly UCI HAR Dataset/test/Inertial Signals for test. The script reads each .txt file and populates a [7352, 128, 6] numpy matrix. Now, originally, the reference didn't provide for a val split. So, the script also finds out which rows in the populated matrix are from which subject by referring to UCI HAR Dataset/train/subject_train.txt, and then forms a partition for a validation set. Thereafter, the matrices are normalised and shuffled and saved.

**Comment by metastableB** [comment]
LGTM. Merging.

**Pull request merged**